From 2c73350073ae64e9a3b4fcf4be81380186b894b8 Mon Sep 17 00:00:00 2001
From: 001183643 <zeinab.ziabari@student.tafesa.edu.au>
Date: Wed, 20 Mar 2024 20:01:43 +1030
Subject: [PATCH 4/4] More discount

---
 Ticketing/Ticketing/TicketPrice.cs | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Ticketing/Ticketing/TicketPrice.cs b/Ticketing/Ticketing/TicketPrice.cs
index ebde21c..7dfb574 100644
--- a/Ticketing/Ticketing/TicketPrice.cs
+++ b/Ticketing/Ticketing/TicketPrice.cs
@@ -14,9 +14,9 @@ namespace Ticketing
         private decimal amountDue;
         private decimal mPrice;
 
-        const decimal mdecBalcony = 40.5m;
-        const decimal mdecGeneral = 30.75m;
-        const decimal mdecBox = 62.0m;
+        const decimal mdecBalcony = 30.5m;
+        const decimal mdecGeneral = 20.75m;
+        const decimal mdecBox = 50.0m;
         const decimal mdecDiscount = 5.0m; 
 
         private int Section
-- 
2.43.0.windows.1

